<launch>
	
	<node name="pr2_head_manager" pkg="pr2_head_manager" type="pr2_head_manager" respawn="false" output="screen" />
	<node name="head_scan" pkg="dt_head_gestures" type="head_scan_node" respawn="false" output="screen" />
    <include file="$(find mocap_filter_head_manager)/launch/mocap_pr2_human_head.launch" />

</launch>
